libs/blocks/editorial-card/editorial-card.js

```diff
@@ -0,0 +1,128 @@
+import { createTag, loadStyle, getConfig } from '../../utils/utils.js';
+import { decorateBlockBg, decorateBlockText, decorateBlockHrs, decorateTextOverrides, applyHoverPlay } from '../../utils/decorate.js';
+
+const { miloLibs, codeRoot } = getConfig();
+const base = miloLibs || codeRoot;
+
+async function loadIconography() {
+  try {
+    const stylePromise = new Promise((resolve) => {
+      loadStyle(`${base}/styles/iconography.css`, resolve);
+    });
+    await Promise.all([stylePromise]);
+  } catch (err) {
+    window.lana?.log(`Failed to load iconography module: ${err}`);
+  }
+}
```

Comment:
We don't have any error handling for loading styles in most of the blocks, hence I'd just go with 
`await new Promise((resolve) => loadStyle(`${base}/styles/iconography.css`, resolve));`